app/src/main/res/layout/activity_scoreboard.xml

```diff
@@ -16,10 +16,10 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent">
 
-        <TableLayout
-            android:id="@+id/scoreTable"
+        <androidx.recyclerview.widget.RecyclerView
             android:layout_width="match_parent"
-            android:layout_height="wrap_content" />
+            android:id="@+id/recyclerViewScoreboard"
+            android:layout_height="match_parent"/>
```

Comment:
Lint warning:
"This androidx.recyclerview.widget.RecyclerView should use `android:layout_height="wrap_content"`"
It seems he is unhappy that you use "match_parent" but I guess the result will look a bit different if you change it?